libs/embed/src/embed.ts

```diff
@@ -143,7 +143,12 @@ class Novu {
         const elem = document.querySelector('.wrapper-novu-widget') as HTMLBodyElement;
 
         if (elem) {
-          elem.style.display = 'inline-block';
+          const elemNotVisible = elem.style.display === 'none';
+          if (elemNotVisible) {
+            elem.style.display = 'inline-block';
+          } else {
+            hideWidget();
+          }
```

Comment:
to simplify the if statements more, you can remove the above one `if (elem)` and instead have this:

```suggestion
        const isWidgetHidden = elem && elem.style.display === 'none';
        if (isWidgetHidden) {
          elem.style.display = 'inline-block';
        } else {
          hideWidget();
        }
```

Reply:
This has been implemented @LetItRock, thanks for the review.